packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -2,21 +2,26 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+import React from 'react';
 import cx from 'classnames';
-import * as React from 'react';
 import {
+  useSafeContext,
+  Box,
+  polymorphic,
+  useIsClient,
+  useIsomorphicLayoutEffect,
   useMergedRefs,
-  getBoundedValue,
   useContainerWidth,
-  useIsomorphicLayoutEffect,
-  useIsClient,
   useResizeObserver,
-  Box,
+  ButtonBase,
 } from '../utils/index.js';
-import { Tab } from './Tab.js';
+import type { PolymorphicForwardRefComponent } from '../utils/index.js';
 import styles from '../../styles.js';
 
-export type OverflowOptions = {
```

Comment:
Any reason we are not exporting `OverflowOptions` anymore?

Reply:
I think we are not exporting any types. So not exporting this one too.